public/index.html

```diff
@@ -84,7 +84,7 @@ <h5 class="card-title">Marketing</h5>
                                 <div class="col-md-6" id="totalMarketingTask"></div>
                             </div>
                         </div>
-                    </div>
+                    </span>
```

Comment:
Please fix the span back to div this might create some problems with small screen, and type button is not needed it will work without that.